app/whiteboard/page.tsx

```diff
@@ -0,0 +1,15 @@
+"use client";
```

Comment:
I'm not 100% sure if `"use client"` is needed here (I just don't know enough about server/client components yet), try removing it and if it works, remove the line. Otherwise ignore this

Reply:
I tried to run it without it and came back with an error that's why.